Core/include/Acts/Surfaces/detail/IntersectionHelper2D.hpp

```diff
@@ -8,11 +8,11 @@
 
 #pragma once
 
-#include <utility>
-
 #include "Acts/Utilities/Definitions.hpp"
 #include "Acts/Utilities/Intersection.hpp"
 
+#include <utility>
```

Comment:
is this reordering (also in other files) due to the different clang-format or is this on purpose?